src/app/components/DesktopHeaderNav/MainNav.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from "react";
+import Link from "next/link";
+import { MainNavProps } from "./DesktopHeaderNav.types";
+import { NavLabel } from "../Header/Header.types";
+
+function MainNav({ navLabels, isActiveLink }: MainNavProps): ReactElement {
+  return (
+    <ul className={` hidden gap-x-10 py-4  md:flex `}>
+      {navLabels.map(({ label, labelIndex, linkPathname }: NavLabel) => (
+        <li key={labelIndex}>
+          <Link
+            href={linkPathname}
+            className={` 
```

Comment:
we have clsx library installed so to combine custom class names with tailwind or to apply some classes conditionally it's a good option